intern_program/intern_program_fall_2018_projects.md

```diff
@@ -34,3 +34,15 @@ This includes building a `.json` catalog of the existing datasets, integration a
 
 #### Skills Required
 Python programming, basic knowledge of html
+
+### Studying melting over Greenland and Antarctica at high spatial resolution
+
+#### Supervisor
+[Marco Tedesco], Lamont Research Professor
+
+#### Description
+
+Studying melting over the Greenland and Antarctica ice sheets is fundamental for understanding their trends and implications for sea level rise. this can be done through the use of passive microwave observations from space, usually delivered at a spatial resolution of 25 km. A new product, however, produces passive microwave data at 3.3 km, hence expanding the possibility of detecting spatial changes or maps of melting but increasing the volume of data to be handled, with a total size of ~ 28 Tb for the period 1979 - 2017. We are looking for someone who will help with the extraction and analysis of the PMW datasets and the generation of melting trends over both ice sheets using tools such as Pangeo or similar.
```

Comment:
Would it be possible to be a bit more specific about the exact objective? In our experience last semester, the interns do best with very clearly defined goals. What would be the final "deliverable" here? Do you envision that the intern would actually conduct research? Or is the goal to create a new database for others to use in their research?